Homework2/StackCalculator/StackCalculator/StackCalculator.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace StackCalculator;
+
+/// <summary>
+/// Represents a calculator that works with reverse polish notation.
+/// </summary>
+public class StackCalculator
+{
+    private IStack _stack;
+    
+    public StackCalculator(IStack stack)
+    {
+        _stack = stack;
+    }
```

Comment:
```suggestion
    public StackCalculator(IStack stack)
        => _stack = stack;
```
Почему нет, это метод-однострочник. Если появится что-то ещё, всегда можно развернуть обратно в форму с фигурными скобками.